normandy/recipes/tests/api/v3/test_api.py

```diff
@@ -218,6 +218,45 @@ def test_list_can_filter_by_filter_object_fields(self, api_client):
             assert res.status_code == 200
             assert res.data["count"] == 0
 
+        def test_list_can_filter_by_filter_object_type(self, api_client):
+            locale = LocaleFactory()
+            recipe1 = RecipeFactory(
+                filter_object=[
+                    filter_objects.BucketSampleFilter.create(
+                        start=100,
+                        count=200,
+                        total=10_000,
+                        input=["normandy.userId", '"global-v4'],
```

Comment:
super tiny, there's a `"` missing after v4?

Reply:
Ah, good catch. I'll fix that.